src/test/java/liquibase/ext/spanner/AddDropDefaultValueComputedTest.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package liquibase.ext.spanner;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.spanner.MockSpannerServiceImpl;
+import com.google.cloud.spanner.Statement;
+import com.google.common.collect.ImmutableList;
+import com.google.spanner.admin.database.v1.UpdateDatabaseDdlRequest;
+import liquibase.Contexts;
+import liquibase.Liquibase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import java.sql.Connection;
+
+@Execution(ExecutionMode.SAME_THREAD)
+public class AddDropDefaultValueComputedTest extends  AbstractMockServerTest {
```

Comment:
`Default` column values and `Computed` column values are different things. This change is specifically for `Default` column values.

```suggestion
public class AddDropDefaultValueTest extends  AbstractMockServerTest {
```

Reply:
(Remove the `Computed` part of the name from the other test files and methods as well to prevent confusion.)

Reply:
change to  AddDropDefaultValueTest and added tests for all type default values